exchange/exchange_brokerage.h

```diff
@@ -23,7 +23,8 @@ class Brokerage {
     std::map<std::string, double> accounts_under_management(const std::string& reporting_ccy) const
     {
         std::map<std::string, double> report;
-        for (const auto& [_, account]: accounts) {
+        for (auto& account_entry: accounts) {
```

Comment:
I was so sad that in C++ 17 we can't do this without a warning that I didn't remove the warning myself. Thank you.